data/2mass_j12451416-4429077.json

```diff
@@ -243,6 +246,20 @@
             "wavelength_order": null,
             "comments": null,
             "reference": "Loop07b"
+        },
+        {
+            "spectrum": "\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000",
```

Comment:
Uh-oh. This is a problem. Not sure what's going on here. What's the spectrum supposed to be?